sdk/identity/identity/src/tokenCache/requireMsalNodeExtensions.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// TODO:
+// Either drop Node 8 or re-enable it.
+
+/**
+ * @internal
+ */
+export const isNode8 = process.versions.node[0] === "8";
+
+/**
+ * @internal
+ */
+export const Node8NotSupportedError = new Error("Node 8 does not support persistence caching.");
+
+/**
+ * @internal
+ */
+export function requireMsalNodeExtensions(): any {
```

Comment:
`any` is unfortunate - maybe we can either find the type through inference and add it here (I get that eslint will require explicit typing here which is probably why we had to add something) or create an internal type that encompasses the things we need / expect to pull out of this module? Not sure, I know there's some complexity there so up to you!

Reply:
I'm unconvinced on doing this up-front since we most likely need to drop Node 8 support this month across the SDK. If we do so, we will be able to remove all of this boilerplate.